docs/PageFormatAndOrientation.md

```diff
@@ -89,3 +89,29 @@ pdf = FPDF()
 pdf.page_mode = "FULL_SCREEN"
 pdf.output("full-screen.pdf")
 ```
+
+
+## skew ##
```

Comment:
The documentation and the examples are great! Good job 😃 

Could you move this section to a new `docs/Transformations.md` file please?
We will also add a section dedicated to rotations, as this function is currently not documented.

Also, could you add this new documentation in the `Graphics Content` section in `mkdocs.yml` please?

Reply:
done